packages/components/src/button/README.md

```diff
@@ -6,9 +6,18 @@ Buttons let users take actions and make choices with a single click or tap.
 
 ## Table of contents
 
-1. [Design guidelines](#design-guidelines)
-2. [Development guidelines](#development-guidelines)
-3. [Related components](#related-components)
+1. [Storybook tool](#storybook-tool)
+2. [Design guidelines](#design-guidelines)
+3. [Development guidelines](#development-guidelines)
+4. [Related components](#related-components)
+
+## Storybook tool
```

Comment:
My only concern here is the vertical space and repetitive content added to the top of every component documentation page. 

That said, I think this looks fine as it fits within the existing structure of the pages.